confidant/authnz/userauth.py

```diff
@@ -131,6 +132,12 @@ def current_first_name(self):
     def current_last_name(self):
         return self.current_user()['last_name']
 
+    def current_role(self):
+        if type(self.current_user()['role']) is list:
```

Comment:
Doing a type check here seems odd. Shouldn't this be: `if not self.current_user()['role']:`

Reply:
fixed